### DISCUSSION TITLE
FindTransferAnchors for ~2 million cells

### DISCUSSION THREAD

**Discussion by ahyeonhwang** [satijalab/seurat#5705] [Q&A]
Hello,

I am currently trying to integrate around 1.6 million RNA cells and 440K ATAC cells with the FindTransferAnchors function but run into a memory issue: Error: cannot allocate vector of size 5874.0 Gb.

I have subset to just 200 highly variables genes and still have this issue. What is the minimum number of genes that I should use for integration?

Do you have recommendations on how to resolve this problem? Do you know of any other RNA+ATAC integration softwares that can deal with large data?

Thank you very much for your help!

Best,
Ahyeon

**Comment by timoast** [comment]
Can you show the code you're running and the output of `sessionInfo()`?
